ha/core/cluster/cluster_manager.py

```diff
@@ -180,9 +187,101 @@ def add_node(self, node):
         _output, _err, _rc = self._execute.run_cmd(const.PCS_STATUS)
         Log.info(f"Cluster status output after add node: {_output}, {_err}, {_rc}")
 
+    def get_nodes_status(self):
+        """
+        Sample output of the const.PCS_STATUS_NODES command
+
+        Pacemaker Nodes:
+         Online: node1 node2
+         Standby:
+         Standby with resource(s) running:
+         Maintenance:
+         Offline:
+        Pacemaker Remote Nodes:
+         Online:
+         Standby:
+         Standby with resource(s) running:
+         Maintenance:
+         Offline:
+        """
+        _output, _err, _rc = self._execute.run_cmd(const.PCS_STATUS_NODES, check_error=False)
+
+        self.active_nodes = self.standby_nodes = self.offline_nodes = "false"
```

Comment:
"false" and "true" should be boolean. Please open a github issue and fix it next PR. Mention the github issue here.

Reply:
https://jts.seagate.com/browse/EOS-17121  created for the same